bin/bootstrap.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Installs top level dependencies, installs package dependencies for
+# `lib-react-components` and builds the library, then bootstraps the remaining
+# packages.
+
+ROOT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && cd .. && pwd)"
+LERNA=$ROOT_DIR/node_modules/.bin/lerna
+
+printf 'Installing root dependencies...'
+(cd $ROOT_DIR && npm install)
+
+printf 'Bootstrapping `lib-react-components`...'
```

Comment:
I think the grommet theme will need to build first, then react components, then auth, then the rest. We could consider dropping the babel requirement with the theme so that it's using common js instead of ES imports like the panoptes.js lib so that we don't have to build it.

Reply:
I totally forgot about the grommet theme needing to be built. I'll add it to the script, and we can make a decision on the babel requirement in a separate PR for now.